agent.go

```diff
@@ -1210,3 +1222,51 @@ func winsizeCb(pod *pod, data []byte) error {
 
 	return nil
 }
+
+// Get the agent configuration from kernel cmdline
+func getConfig() error {
+	kernelCmdline, err := ioutil.ReadFile(kernelCmdlineFile)
+	if err != nil {
+		return err
+	}
+	if err := parseKernelCmdline(string(kernelCmdline)); err != nil {
+		return err
+	}
+	return nil
+}
+
+//Parse a string that represents a kernel cmdline to get agent options
+func parseKernelCmdline(cmdline string) error {
+
+	const valuePosition = 1
+	const optionSeparator = "="
+
+	words := strings.Fields(cmdline)
+
+	for _, w := range words {
+		for o := range agentOptions {
```

Comment:
Hi @sboeuf , I updated the PR:
0. I created agentConfig struct (by now we only have logLevel)
1. getConfig() return parse the kernel cmdline 
- If could not parse an option I will use a default option.
2. applyConfig(config) will take a config and apply the values already parsed and validated in getConfig()